script/gen-filenames.js

```diff
@@ -10,13 +13,92 @@ const allDocs = fs.readdirSync(path.resolve(__dirname, '../docs/api'))
       .map(doc => `docs/api/structures/${doc}`)
   )
 
-fs.writeFileSync(
-  gniPath,
-  `# THIS FILE IS AUTO-GENERATED, PLEASE DO NOT EDIT BY HAND
+const main = async () => {
+  const browserifyTargets = [
+    {
+      name: 'sandbox',
+      entry: 'lib/sandboxed_renderer/init.js'
+    },
+    {
+      name: 'isolated',
+      entry: 'lib/isolated_renderer/init.js'
+    },
+    {
+      name: 'context_script',
+      entry: 'lib/content_script/init.js'
+    }
+  ]
```

Comment:
i'd love it if `context_script_browserify_deps` as mentioned in `filenames.gni` was greppable in this file. can we put the entire variable name here rather than just the bit before `_browserify_deps`?